benchmark/mprof.sh

```diff
@@ -0,0 +1,11 @@
+export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
+
+# TODO: provide flag to `bazel run benchmark:memory` to choose between
```

Comment:
Info / for later: Flags can be passed to binaries as follow:
bazel run benchmark:memory -- <flags to pass to the :memory binary>